test/xspec-bat.cmd

```diff
@@ -301,6 +331,30 @@ setlocal
     call :teardown
 endlocal
 
+setlocal
+    call :setup "executing the Saxon XProc harness generates a report with UTF-8 encoding"

```

Comment:
[Dup](https://github.com/vincentml/xspec/blob/de4a2ad60bef1e2b6542882fca95869ca3b1549b/test/xspec-bat.cmd#L290)? Maybe a remnant of conflicts.